demisto_sdk/commands/lint/linter.py

```diff
@@ -232,22 +230,13 @@ def _gather_facts(self, modules: dict) -> bool:
         if self._facts["lint_files"]:
             for lint_file in self._facts["lint_files"]:
                 logger.info(f"{log_prompt} - Lint file {lint_file}")
+            for lint_file_test in self._facts["tests"]:
+                logger.info(f"{log_prompt} - Lint file {lint_file_test} - Only flake8")
         else:
             logger.info(f"{log_prompt} - Lint files not found")
 
-        self._split_lint_files()
```

Comment:
Why did we remove this?

Reply:
Its redundant step as I mentioned above, Flake8 done twice instead of once.